docs/adr/005_observability.md

```diff
@@ -74,24 +74,19 @@ In addition to this, we will support Application Insights metrics on an opt-in b
-| RxWndRate                  | Number of times we hit the different receive windows.        | LNS    | LoRaWan   | Gateway Id, Receive Window |
-| RxWndMiss                  | Number of missed on downstream windows                       | LNS    | LoRaWan   | Gateway Id                 |
+| ReceiveWindowHits | Number of times we hit the different receive windows.        | LNS    | LoRaWan   | Gateway Id, (estimated) Receive Window |
+| ReceiveWindowMisses | Number of missed on downstream windows                       | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceCacheHit             | Number of device cache hit                                   | LNS    | LoRaWan   | Gateway Id                 |
 | DeviceLoadRequests         | Number of device load requests                               | LNS    | LoRaWan   | Gateway Id                 |
 | JoinRequests               | Number of join requests                                      | LNS    | LoRaWan   | Gateway Id                 |
 | StationConnectivityLost    | Connection to LBS lost                                       | LNS    | LoRaWan   | Gateway Id                 |
 | ActiveStationConnections   | Active connections to stations                               | LNS    | LoRaWan   | Gateway Id                 |
 | D2CMessagesReceived        | Number of messages received from device                      | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessageDeliveryLatency  | Time from when we received the message from the concentrator until we are done processing it | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessagesDelivered       | Number of messages sent to upstream                          | LNS    | LoRaWan   | Gateway Id, To             |
-| D2CMessagesError           | Number of errors in sending messages to upstream             | LNS    | LoRaWan   | Gateway Id, To            |
-| D2CMessagesProcessingError | Number of errors processing (decoding, decrypting) messages  | LNS    | LoRaWan   | Gateway Id, To            |
+| DataMessageDeliveryLatency | Time from when we dispatched the message sent from the concentrator until we are done processing it | LNS    | LoRaWan   | Gateway Id                 |
+| DataMessageDispatchLatency | Time from when we received the message from the concentrator until we dispatch the message to be handled (internal metric) | LNS | LoRaWan | Gateway Id, Message Type |
 | D2CMessageSize             | Message size in bytes received from device                   | LNS    | LoRaWan   | Gateway Id                 |
-| D2CMessageSizeUpstream     | Message size in bytes sent upstream                          | LNS    | LoRaWan   | Gateway Id, To            |
 | C2DMessagesAbandoned       | Number of C2D messages abandoned                             | LNS    | LoRaWan   | Gateway Id                 |
```

Comment:
[Brainstorming] Would it be something to track the signal strength (RSSI) and signal to noise ratio (RSSI)

Reply:
What would you use it for? Optimize the deployment of the concentrators? You would need to know that devices have a low RSSI, no? Tracking the device id as dimension, is probably not possible due to the high cardinality.